combined.js

```diff
@@ -415,20 +420,40 @@ function drawCanvas(canvas, img, draw_type)
     }
 }
 
-/*
-    TODO: Base function for TFJS based draws.
-    Should handle BodyPix variations (Overlay/Pixels) and Pose-Animator.
-*/
-async function tensor_draw_pixel(canvas, img)
+async function segmentGenerator(video)
 {
-    try{   
```

Comment:
Splitting up segment Generation and drawing into 2 bits.
segGen is the most compute intensive part.